app/views/content/early-years-teaching-training.md

```diff
@@ -55,33 +55,35 @@ calls_to_action:
         Salary: "No"
         Funding available: "No"
         Length of training: "3 months"
-        Qualification: "Early Years Teacher Status"
-      - "Graduate Assessment-Only training"
+        Qualification: "Early years teacher status"
+      - "Graduate 'assessment only' training"
 ---
 
 ## Become an early years teacher
- 
-As an early years teacher, you'll work with young children up to the age of 5. Your input will have a lasting and positive effect on their development and wellbeing.
- 
-By training to be an early years teacher, you’ll use your knowledge and skills to help young children enjoy high standards of teaching. 
 
-The early years are critical in children's development. You’ll be opening their minds to new ideas every day.
- 
+As an early years teacher you'll work with young children up to the age of 5. Your input will have a lasting and positive effect on their development and wellbeing.
+
+The early years are critical in children's development. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
+
 ###How to become an early years teacher
- 
-You’ll need to achieve Early Years Teacher Status (EYTS). This is different from Qualified Teacher Status (QTS), which is the equivalent for teaching children aged 3 to 18. 
+
+You’ll need to do 'early years initial teacher training' (EYITT) to achieve 'early years teacher status' (EYTS).
+
+EYTS is different from '[qualified teacher status](/ways-to-train)' (QTS), which is the equivalent for teaching children aged 3 to 18.
 
 Your training will enable you to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
- 
+
 To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
 
-* Maths grade 4 (C)
+* maths grade 4 (C)
 * English grade 4 (C)
-* Science grade 4 (C)
+* science grade 4 (C)
```

Comment:
Aren't subject names proper nouns?

Reply:
![image](https://user-images.githubusercontent.com/35870975/118509734-39217180-b728-11eb-8c48-9f1cc033cf0e.png)

I believe it's only English

Reply:
on Apply for teacher training we user lower case for everything except English, and on gov.uk you'll see a lot of lower case going on e.g. https://www.gov.uk/government/publications/gcse-single-science